__tests__/app.test.js

```diff
@@ -66,3 +66,30 @@ describe("getArticleById", () => {
       });
   });
 });
+describe("updateVotes", () => {
+  test("status: 200 and responds with the updated votes in the article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .expect(200)
+      .send({ inc_votes: 1 })
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(101);
+      });
+  });
+  test("status:404 sends error message when given a valid but non-existent address", () => {
```

Comment:
you don't need to do the test for every endpoint. A typo in this endpoint is the same as going to `/bananas`